modules/ROOT/pages/_partials/sync/ref_terminology.adoc

```diff
@@ -1,14 +1,16 @@
-//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
-[id='ref_terminology_{context}']
+[id='ref-terminology-{context}']
 
 = {sync-service} Terminology
 
-This section describes terminology that is associated with {sync-service}.
+This section describes terminology that is associated with Data Sync.
 
-GraphQL:: A query language for your API, and a server-side runtime for executing queries by using a type system you define for your data. An alternative approach to REST for building APIs and applications. See link:https://graphql.org/learn[GraphQL].
+Data Sync framework - Data Sync is a JavaScript framework that enables a developer to add real time data synchronization to both mobile and web clients.
```

Comment:
Does it add the capability to synchronize data in real time or does it synchronize in real time. Also, if you are using rea time as an adjective, it needs to be hyphenated.

Reply:
```suggestion
Data Sync framework - Data Sync is a JavaScript framework that enables a developer to add the capability to synchronize data in real-time for both mobile and web clients.
```

Reply:
@pnagesha wdyt ^